classifiers/query_check/communication_style_classifier/__init__.py

```diff
@@ -0,0 +1,27 @@
+from pydantic import BaseModel
+import requests
+
+INPUT_EXAMPLE = {
+    "text": "Change the number in row 2 and 3.",
+    "model_name": "KernAI/multilingual-e5-communication-style",
+}
+
+
+class CommunicationStyleClassifierModel(BaseModel):
+    text: str
+    model_name: str
+
+    class Config:
+        schema_extra = {"example": INPUT_EXAMPLE}
+
+
+def communication_style_classifier(req: CommunicationStyleClassifierModel):
+    """Uses custom E5 model to classify communication style of a text"""
+    payload = {
+        "name_model": req.model_name,
```

Comment:
We should name the parameters in the same way. Personally, I also like `model_name` more, I tried it for hosted inference API but received a warning there, so I changed it. I'll check again if we can use `model_name` there as well.

Reply:
Perfect, changed it!